lib/Condition.ts

```diff
@@ -1,19 +1,19 @@
-import {Document} from "./Document";
+import { Document } from "./Document";
 import CustomError = require("./Error");
 import utils = require("./utils");
 const OR = Symbol("OR");
-import {DynamoDB} from "aws-sdk";
+import { DynamoDB } from "aws-sdk";
```

Comment:
It'd be great if there was a lint option to make this uniform... anyone happen to know of any ways to enforce this in the linter?

Reply:
Just add tslint to the project